guides/installation_guides/c++/mac/README.md

```diff
@@ -1,7 +1,35 @@
-# Cosmos Guides
-> Your personal library of every algorithm and data structure code that you will ever encounter
+# Cosmos Guide: Getting Ready to program in C++ in MAC OS
+ **<b>Getting The Software</b>** <br> <br>
+ There are many IDE's for MAC OS but here's a quick guide to install one of them i.e <b> CODEBLOCKS. <br><br>
+ You can get Codeblocks from the Official Site: http://www.codeblocks.org/downloads <br>
+Click on <b>Download the binary release</b> <br>
+ <br> <br>
 
+Look after codeblocks's <b>mingw-setup.exe</b>, it may be a different version than the one in the screenshot but it does not matter. Click on the <b>SourceForge</b><br>
```

Comment:
Er, you probably want to move this to a separate PR.

Reply:
sure 😄